modules/microshift-viewing-audit-logs.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+//microshift_troubleshooting/microshift-audit-logs.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-security-context-constraints-alert-eval_{context}"]
+= Identifying pod security violations through audit logs
+
+You can identify pod security admission violations on a workload by viewing the server audit logs. The following procedure shows you how to access the audit logs and parse them to find pod security admission violation in a workload.
+
+.Prerequisites
+
+* You have installed `jq`.
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. To view the audit logs, run the following command: 
++
+[source,terminal]
+----
+$ NODE_NAME=$(oc get node -ojsonpath='{.items[0].metadata.name}')
+----
++
+[source,terminal]
+----
+$ oc adm node-logs "${NODE_NAME}" --path=kube-apiserver/
+----
++
+.Example output
+[source,terminal]
+----
+rhel-92.lab.local audit-2023-08-18T18-25-41.663.log
+rhel-92.lab.local audit-2023-08-19T11-21-29.225.log
+rhel-92.lab.local audit-2023-08-20T04-16-09.622.log
+rhel-92.lab.local audit-2023-08-20T21-11-41.163.log
+rhel-92.lab.local audit-2023-08-21T14-06-10.402.log
+rhel-92.lab.local audit-2023-08-22T06-35-10.392.log
+rhel-92.lab.local audit-2023-08-22T23-26-27.667.log
+rhel-92.lab.local audit-2023-08-23T16-52-15.456.log
+rhel-92.lab.local audit-2023-08-24T07-31-55.238.log
+----
```

Comment:
For me i did not see the machine name like above `rhel-92.lab.local` but rest looks good to me.